src/openforms/authentication/contrib/org_oidc/backends.py

```diff
@@ -0,0 +1,18 @@
+import logging
+
+from mozilla_django_oidc_db.backends import (
+    OIDCAuthenticationBackend as _OIDCAuthenticationBackend,
+)
+
+from .mixins import SoloConfigMixin
+
+logger = logging.getLogger(__name__)
+
+
+class OIDCAuthenticationBackend(SoloConfigMixin, _OIDCAuthenticationBackend):
+    config_identifier_field = "username_claim"
```

Comment:
I can't immediately tell if this is used / different from the default - is this still needed?